deploy/crds/openebs.io_jivavolumepolicies.yaml

```diff
@@ -1484,10 +1486,9 @@ spec:
                           type: string
                       type: object
                     type: array
+                required:
+                - monitor
```

Comment:
We have to fix the JivaVolumePolicy CR yaml  as well,  since 
```
the JivaVolumePolicy "example-jivavolumepolicy" is invalid: spec.target.monitor: Required value
```

Reply:
The CR yaml should have generated , did you missed to add in PR ?

Reply:
No, it was not generated. 
`make manifests` only updated both the crds yaml.

Reply:
yes , but we need to update the policy CR with above mentioned changes.

Reply:
Done.